app/code/Magento/Sales/Test/Mftf/ActionGroup/AdminSelectFixedShippingMethodActionGroup.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminSelectFixedShippingMethodActionGroup">
```

Comment:
It might be a little bit confusing how exactly this action group selects the "Fixed" shipping method. Maybe it makes sense to name it as `AdminSelectFirstShippingMethodActionGroup`. What do you think?

Reply:
Hi @rogyar 
This Action Group select` Fixed `Shipping Method  not First
Due to this element 
`<element name="shippingMethod" type="button" selector="//label[contains(text(), 'Fixed')]" timeout="60"/>`
https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/Sales/Test/Mftf/Section/AdminInvoicePaymentShippingSection.xml#L19

Reply:
Got it, thanks